File: src/Application/ClientApp/app/models/user-model.ts
export enum UserStatus
{
    Active,
    Disabled,
    Blocked
}

export class UserModel {
    id: number;
    email: string;
    phone: string;
    firstName: string;
    lastName: string;
    createdDate: Date;
    name: string;
    role: string;
    status: UserStatus;
}

File: src/Entities/Models/UserModel.cs
using Application.EntitiesModels.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text;

namespace Application.EntitiesModels.Models
{
    public class UserModel
    {
        public int Id { get; set; }

        [Required]
        [EmailAddress]
        [Display(Name = "Email")]
        public string Email { get; set; }

        //Test
        [Required]
        [DataType(DataType.Password)]
        [Display(Name = "Пароль")]
        public string Password { get; set; }
        public string Phone { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime CreatedDate { get; set; }
        public string Name { get; set; }
        public string Role { get; set; }
        public UserStatus Status { get; set; }

        public IEnumerable<string> Logins { get; set; }
    }
}
